src/Account/HistoryClient.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace UKFast\SDK\Account;
+
+use UKFast\SDK\Account\Entities\History;
+use UKFast\SDK\Client as BaseClient;
+
+class HistoryClient extends BaseClient
+{
+    protected $basePath = 'account/';
+
```

Comment:
As we now have to map, we'll need a request map

```suggestion
    const MAP = [
        'id'         => 'id',
        'contact_id' => 'contactId',
        'username'   => 'username',
        'date'       => 'date',
        'reason'     => 'reason',
        'ip'         => 'ip',
        'url'        => 'url',
        'useragent'  => 'userAgent',
    ];

```